### DISCUSSION TITLE
What about .Jar games emulation?

### DISCUSSION THREAD

**Discussion by ThePitFighter** [SquirrelJME/SquirrelJME#174] [RetroArch (RatufaCoat)]
Hello, found out your core in PSV retroarch and I have seen few questions about current state of Jar emulation.
So i have 2 questions:
1. What does your **Retroarch Core** can emulate right now?
2. Is it still in progress and do you have any estimated time for solving this problem?

**Comment by XerTheSquirrel** [comment]
Hi!

The core is still in progress yes, but what exists in RetroArch cannot run anything yet. It is still in progress yes and I hope to have something by this year, hopefully September as long as everything lines up and nothing out of the ordinary happens.
